source/code/plugins/filter_patch_management.rb

```diff
@@ -31,7 +31,8 @@ def shutdown
     def filter(tag, time, record)
       xml_string = record['xml']
       @log.debug "LinuxUpdates : Filtering xml size=#{xml_string.size}"
-      return LinuxUpdates.transform_and_wrap(xml_string, @hostname, time)
+      linuxUpdates = LinuxUpdates.new(@log, nil)
```

Comment:
Because the second parameter is current_update_run_file, it is a config parameter for filter_linux_update_run_progress. Not apply to this filter. So pass nil here.